src/get-workspaces-paths.js

```diff
@@ -31,51 +35,148 @@ function processYarn({ stdout }) {
   return workspaces;
 }
 
+function processGlobs(cwd, _2dFilesArray){
+  let _1dFilesArray = Array.prototype.concat.apply([], _2dFilesArray);
+
+  let uniqueFiles = [...new Set(_1dFilesArray)];
+
+  let packagePaths = uniqueFiles.map(file => path.join(cwd, file, 'package.json'));
```

Comment:
You should not include package.json here, instead using it as needed. Then you won't have to remove it later with `.replace('/package.json', '')`